src/main/java/org/jitsi/videobridge/Videobridge.java

```diff
@@ -263,7 +289,12 @@ public Conference createConference(
 
         do
         {
-            String id = generateConferenceID();
+            String id;
+            if(useNameForConferenceId && !StringUtils.isNullOrEmpty(name)) {
+                id = name;
+            } else {
+                id = generateConferenceID();
+            }
```

Comment:
No preference. Looks like anything will be OK for the XMPP part -- throwing any exception or returning null from createConference() or createRtpChannel() should result in an error IQ being returned.